### PR TITLE
[WIP]Add notifications to releases

### PR DESCRIPTION
**Pull request by Bwko**
Adds #640

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
but need a release notify mail template?

**Comment by andreynering** [comment]
Looking at the code, seems that it will only create an entry in the feed, but not an notification? I think that was the intention of the linked issue? @lunny

**Comment by Bwko** [comment]
It's still a WIP, I still need to add the real notification

File: routers/repo/release.go [hunk 1]
```diff
@@ -169,7 +169,7 @@ func NewRelease(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("repo.release.new_release")
 	ctx.Data["PageIsReleaseList"] = true
 	ctx.Data["tag_target"] = ctx.Repo.Repository.DefaultBranch
-	renderAttachmentSettings(ctx);
+	renderAttachmentSettings(ctx)
```
Comment (ethantkoenig) [inline routers/repo/release.go:172]: :smiley:

**Comment by lunny** [comment]
@Bwko any news?

**Comment by bkcsoft** [comment]
How about we not L-G-T-M on WIP stuff? 😛

**Comment by lunny** [comment]
I think this could be two PRs. One is for feed, another is for notification.

**Comment by bkcsoft** [comment]
@lunny IMO it makes sense to have those as 1 PR

**Comment by lunny** [comment]
Let's move this to v1.2

**Comment by bkcsoft** [comment]
I cleaned out L-G-T-Ms since this is still a WIP 🙂

**Comment by lunny** [comment]
any update? @Bwko

**Comment by bkcsoft** [comment]
Closing as dead. Feel free to send another PR if you pick this up again. (This is so people won't wait on something that isn't coming)

**Pull request closed without merging**